.gitignore

```diff
@@ -36,6 +36,7 @@
 /rabbitmq-broker
 /redis-broker
 /s3
+/shieldp
```

Comment:
What is `shieldp`?

Reply:
This was the shield plugin binary. I'll remove it from this pr.